test/libsolidity/syntaxTests/nameAndTypeResolution/016_assignment_to_mapping.sol

```diff
@@ -4,10 +4,9 @@ contract test {
     }
     str data;
     function fun() public {
-        var a = data.map;
+        mapping(uint=>uint) a = data.map;
```

Comment:
Should be ``mapping(uint => unit)``.

Reply:
I saw both coding styles, with and without surrounding spaces around `=>`. I prefer the former, but I adapted to the coding style of the original author here (line 3). (is your `unit` a typo here?)